wasm/src/graph_data.rs

```diff
@@ -79,21 +80,20 @@ impl EdgeData {
         // the mapping that exist on the JS side are as follows
         // "field" | "explicit" | "source" | "implied_kind" | "round"
 
-        // TODO: maybe change the attribute options so that the JS side better matches the data
-
+        // TODO(JS): maybe change the attribute options so that the JS side better matches the data
```

Comment:
I could've named it better, but with the `TODO(RUST)` comments, I more just meant TODOs left for this PR, not _where_ the todo needs to be done :)